.github/workflows/continuous-integration.yaml

```diff
@@ -2,7 +2,7 @@ name: 'Continuous integration'
 on: ['push', 'pull_request']
 jobs:
   cs:
-    runs-on: 'ubuntu-20.04'
+    runs-on: 'ubuntu-22.04'
```

Comment:
24.04 maybe?

Reply:
Trying it out. But the error seems to be something else. I'm going to skip the broken test for now as I'm pretty sure it is not related.